### ISSUE TITLE
[Feature Request] Cancel reply by pressing ESC

### ISSUE THREAD

**Issue by crawlserv** [NicolasConstant/sengi#461]
Hey!
One tiny feature I am missing in Sengi compared to other Desktop clients is the ability to quickly cancel a reply by pressing ESC after I accidentally pressed "Reply" on a toot. It took a while to realize that I had to press the "Reply" button on that toot again. (Sorry for the multiple requests but I really enjoy using Sengi:)

**Comment by NicolasConstant** [comment]
(no issues, and thanks for you're contribution! glad you like the app!)

**Comment by teddyber** [comment]
yes ESC to either cancel reply or to leave focus of the "new status" input box would be neat (and then "c" to close the left bar)
